### ISSUE TITLE
NullPointerException, if call "startLogin" method

### ISSUE THREAD

**Issue by igorbljahhin** [codeborne/mobileid#8]
The method com.codeborne.security.mobileid.MobileIDAuthenticator#startLogin(java.lang.String, java.lang.String) is always failing with NPE:

```java
Caused by: java.lang.NullPointerException: null
	at com.codeborne.security.mobileid.MobileIDAuthenticator.startLogin(MobileIDAuthenticator.java:122) ~[mobileid-1.1.jar:na]
	at com.codeborne.security.mobileid.MobileIDAuthenticator.startLogin(MobileIDAuthenticator.java:103) ~[mobileid-1.1.jar:na]
```

**Comment by asolntsev** [comment]
@igorbljahhin Wow. Thank you for reporting. I will take a look.

**Comment by igorbljahhin** [comment]
No, the method com.codeborne.security.mobileid.MobileIDAuthenticator#startLogin(java.lang.String, java.lang.String, java.lang.String) is protected, I can't call it. I call com.codeborne.security.mobileid.MobileIDAuthenticator#startLogin(java.lang.String, java.lang.String):

![image](https://user-images.githubusercontent.com/1252230/33530701-8f759c56-d88c-11e7-9342-7b0c296fb2ae.png)

**Comment by asolntsev** [comment]
Why you need to call method `startLogin(String personalCode, String countryCode)` **without** phone number? Is it really a valid way to use MobleID? Probably we just need to delete this method?

**Comment by igorbljahhin** [comment]
Our task was to implemented login procedure using MobilID by personal identification ID. That's why I tried to use this method. But now, after looking around at other sites, I guess, you are right, phone number should be mandatory, but personal identification ID should be optional.

**Comment by asolntsev** [comment]
The fix has been released as version 1.2:
http://search.maven.org/#search%7Cga%7C1%7Cmobileid
